code/src/layouts/tables/orders-table/components/orders-table-body/OrdersTableBody.scss

```diff
@@ -0,0 +1,15 @@
+@import "@design-system";
+
+.spa-order-table__body {
+  @include transition((background-color));
+
+  background-color: var(--spa-order-table-body-bg-color, $white);
```

Comment:
`bg` write full naming for `background-color`

Reply:
We are using shortcut for background. Look for example AppButton or other app components

Reply:
> We are using shortcut for background. Look for example AppButton or other app components

It was implemented by you
<img width="981" alt="image" src="https://github.com/idx-academy/spa-orders/assets/56305508/784e8dc6-6e6d-4bcb-b0fb-a03adc9d2fb9">

Follow single approach with your naming creating before

Reply:
changed naming of the table-container